recipes/use_cases/text2sql/StructuredLlama.ipynb

```diff
@@ -101,437 +76,146 @@
    "id": "6d421ae7",
    "metadata": {},
    "source": [
-    "Next you will need create the `nba_roster.db` file. \n",
+    "To recreate the `nba_roster.db` file, run the two commands below:\n",
+    "- `python txt2csv.py` to convert the `nba.txt` file to `nba_roster.csv`. The `nba.txt` file was created by scraping the NBA roster info from the web.\n",
```

Comment:
It's not clear where these files are and where the scripts to run them are. Should we point users to the nba_roster.db file in the repo and show them how to upload it to Colab for example? Running these in colab always, maybe that is an issue.

Reply:
Uploaded the provided nba_rooster to colab and was able to make it work till the end!